### PR TITLE
Add Python SDK

### PR DESCRIPTION
**Pull request by mplooster**
Add the autogenerated gRPC python (diatheke_p2b.py and diatheke_p2b_grpc.py) and wrappers around them. Reviewers only need to worry about checking the wrappers and not the autogenerated code.

To make them a little more manageable, this is the first of three PRs related to the Python SDK. The remaining two will include example code and documentation.

### CODE REVIEW

File: grpc/py-diatheke/diatheke/streams.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        self._stream_thread = threading.Thread(target=client_stub.StreamAudioInput,
+                                               args=(self,))
+        self._stream_thread.start()
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        # Note this function is being called from the thread that the actual
+        # stream is running on.
```
Comment (jhollowayj) [inline grpc/py-diatheke/diatheke/streams.py]: Does this mean `self._stream_thread` or a different thread or a different thread?

Comment (mplooster) [reply grpc/py-diatheke/diatheke/streams.py]: Correct. This means `self._stream_thread`.

Comment (jhollowayj) [reply grpc/py-diatheke/diatheke/streams.py]: Would you mind adding that as a comment in the code?

File: grpc/py-diatheke/diatheke/streams.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        # some threading to make this work.
+
+        # Set up a queue to transfer data from the write function to the
+        # __next__ function.
+        self.data_queue = Queue(maxsize=1)
+        self.session_id = session_id
+        self._id_sent = False
+        self._is_finished = False
+
+        # Set up a thread to run the StreamAudioInput function with this class
```
Comment (happyalu) [inline grpc/py-diatheke/diatheke/streams.py]: Could you please elaborate on this a little bit? I'm not sure why we need to use a separate thread here.

**Pull request merged**